docs/product-assembly_decision-manager/src/main/asciidoc/main.adoc

```diff
@@ -0,0 +1,38 @@
+[id='assembly_decision-manager-getting-started']
+
+include::product-shared-docs/document-attributes.adoc[]
+
+= Getting started with decision services
+// Context attribute is assembly specific and enables module reuse between assemblies.
+:imagesdir: topics/product-shared-docs/images
+:context: decision-manager-getting-started
+include::product-shared-docs/author-group.adoc[]
+
+// Purpose statement for the assembly
+As a business analyst or rules developer, you can use {CENTRAL} in {PRODUCT} to design a variety of decision services. In this tutorial, you will create and test a driver's license suspension model by creating a data model that is made up of data objects with data fields, such as Violation Date, Fine Amount, and so on.
+
+.Prerequisites
+* Installed {EAP_LONG} 7.1.0. For details, see  https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.1/html-single/installation_guide/[_Red Hat JBoss EAP 7.1.0 Installation Guide_].
```

Comment:
EAP is installed, DM is installed, etc. The style we're trying to be consistent with. Emily's going to make that explicit in our BXMS SG soon.

Reply:
Hi. Not sure what is missing in my topic. Here is the content of the prereq section:
.Prerequisites
* Installed {EAP_LONG} 7.1.0. For details, see  https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.1/html-single/installation_guide/[_Red Hat JBoss EAP 7.1.0 Installation Guide_].
* Installed {PRODUCT}. For further information, see the {INSTALLATION_GUIDE}.
* {PRODUCT} is running and you can log in to {CENTRAL} with the `admin` role. For further information, see the Installing Decision Manager on premise guide.